Integrations/integration-Wildfire.yml

```diff
@@ -94,11 +102,35 @@ script:
         );
 
         if (res.StatusCode !==200) {
-          throw 'Request Failed.\nStatus code: ' + res.StatusCode +' '+errorDict[res.StatusCode]+ '.\nBody: ' + JSON.stringify(res) + '.';
+            throw 'Request Failed.\nStatus code: ' + res.StatusCode + ' ' + ERROR_DICT[res.StatusCode] + '.\nBody: ' + JSON.stringify(res) + '.';
         }
         return JSON.parse(x2j(res.Body));
     };
 
+    function prettifyUploadBody(body) {
+        var prettyUploadBody = {
+            'MD5': body.md5,
+            'SHA256': body.sha256,
+            'FileType': body.filetype,
+            'Size': body.size,
+            'Status': 'Pending'
+        };
+        return prettyUploadBody;
+    }
+
+    function createUploadEntry(body, title, result) {
+        var md = tableToMarkdown(title, body, Object.keys(body));
```

Comment:
no need to pass headers argument if you are using all the keys

Reply:
true, fixed.